src/main/java/com/o19s/es/explore/StatisticsHelper.java

```diff
@@ -0,0 +1,87 @@
+/*
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.o19s.es.explore;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class StatisticsHelper {
```

Comment:
I think you need to protect against an edge case where there are no points in the data array.
Or add `assert !data.isEmpty();` before all access to data to indicate future devs that this class cannot be used with an empty data array.